Monika After Story/game/script-topics.rpy

```diff
@@ -3264,42 +3264,54 @@ init 5 python:
     )
 
 label monika_justification:
-    #lose affection
-    $ mas_loseAffection(reason=3)
-    if mas_isMoniLove():
-        m 1euc "..."
-        m 1hua "Ahaha! Very funny [player]!"
-        m 3eua "I know you don't really mean it..."
-        m 3rksdlc "But don't joke around like that!"
-        if persistent._mas_pm_a_hater:
-            m 2rksdld "You've said stuff like that before, so it might get harder to take it as a joke..."
-        else:
-            m 1dsc "If I didn't know you so well, that would have been really cruel..."
+
+    if datetime.date.today() == datetime.date(datetime.date.today().year,4,1) and mas_isMoniNormal(higher=True):
```

Comment:
Would likely make a variable in definitions dedicated to april 01. `mas_a01 = datetime.date(datetime.date.today().year, 4, 1)`, then use that constant in this topic.